dotcom-rendering/src/components/BigSixOnwardsContent.tsx

```diff
@@ -126,7 +133,7 @@ export const BigSixOnwardsContent = ({ url, discussionApiUrl }: Props) => {
 								imagePositionOnDesktop="right"
 								imagePositionOnMobile="top"
 								imageSize="large"
-								imageLoading="lazy"
+								imageLoading="eager"
```

Comment:
We only load this component when the user has scrolled to it, so we should load eagerly. Mirrors functionality in Carousel component where the four non-hidden cards images are eagerly loaded.